content/blog/2025-11-18-proof-of-work-smart-contracts-and-global-censorship-rresistance/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Ethereum Classic: Proof-of-Work Smart Contracts and Global Censorship Resistance"
+date: 2025-11-17
+author: Cody Burns
+contributors: ["RealCodyWBurns"]
+tags: ["Mining"]
+linkImage: ./banner.png
+---
+
+![](./banner.png)
+
+Ethereum Classic is currently the largest proof-of-work smart contract platform. The network operates at roughly 300 terahashes per second (TH/s), according to public hashrate trackers such as [2Miners](https://2miners.com/etc-network-hashrate). This represents approximately 90 to 95 percent of all Ethash or Etchash compatible hashing power across all networks.
+
+This level of mining participation has practical implications. It supports a permissionless environment where individuals and organizations anywhere in the world can participate without identification or prior approval. It also strengthens resistance to censorship because mining hardware and miners themselves are geographically dispersed.
+
+
+---
```

Comment:
Can we remove these dividers to maintain formatting consistency with other articles, please?

Otherwise, LGTM

Reply:
ack